firmware/boron-ins-fsm/src/stateMachine.cpp

```diff
@@ -384,6 +393,18 @@ void getHeartbeat() {
         char heartbeatMessage[622] = {0};
         JSONBufferWriter writer(heartbeatMessage, sizeof(heartbeatMessage) - 1);
         writer.beginObject();
+
+        if (missedDoorEventQueue.size() > missedDoorEventTrackN) {
+            if (missedDoorEventQueue.front())
+	        missedDoorEventAmnt--; // oldest value did miss; subtract it from the current amount
```

Comment:
In the `firmware/README.md` file, you'll see a section about "Firmware Code Linting and Formatting". We use the `clang` program to have consistent formatting in our C++ files. I believe that these format rules require opening and closing braces for all blocks (meaning that I think you need to have the `{` and `}` around this inside of this `if`.

It's a good idea to run the clang formatter on your local machine before committing, to catch formatting issues early. However, it's a bit of a picky program and the version matters. So sometimes it's easier just to push anyways and see if the errors come up when Travis automatically runs the linter.

The reason that we decided to make the curly braces required is to reduce the likelihood of error. Since indentation in C++ doesn't matter like it does in Python, it's quite easy to add code aligned to inside the if-block but it is actually outside of it.

Reply:
Thanks! Glad it's passing Travis now.